jerry-core/jerry.cpp

```diff
@@ -1318,6 +1318,17 @@ jerry_api_eval (const jerry_api_char_t *source_p, /**< source code */
 } /* jerry_api_eval */
 
 /**
+ * Perform GC
+ */
+void
```

Comment:
Usually calling GC is a black box, depending on the implementation it might be a deferred call, priority settings change, etc. So keeping it void is perfect for me (otherwise we just restrict ourselves).